DHIS2/cloner/dhis2_clone

```diff
@@ -80,8 +85,10 @@ def get_args():
     add('--no-backups', action='store_true', help="don't make backups")
     add('--no-webapps', action='store_true', help="don't clone the webapps")
     add('--no-db', action='store_true', help="don't clone the database")
+    add('--no-postprocess', action='store_true', help='don\'t execute postprocessing actions')
```

Comment:
All good, but why not use the same formatting as the rest?

```
add('--no-postprocess', action='store_true', help="don't execute postprocessing actions")
```

Related to that, I try to keep things within 80 lines, in case you wonder why the formatting I normally use. But you are more than welcome to break that :)

Reply:
agreed